src/blocks.js

```diff
@@ -98,141 +128,30 @@ export class CachingBlockStore {
   }
 }
 
-/**
- * A blockstore that copes with the dag.haus bucket legacy.
- * Also adapts car block style blockstore api that returns {cid, bytes}
- * to one that returns just the bytes to blend with miniswap api.
- */
 export class DagHausBlockStore {
```

Comment:
nit: would be cool to keep some docs around why this DagHausBlockStore

Reply:
Yeah...it's just not really DAG house anymore because it doesn't do any of what the docs were saying. Maybe I should just rename?